local.py

```diff
@@ -0,0 +1,74 @@
+# If you want to modify this file, I recommend check out docker-taiga-example
+# https://github.com/benhutchins/docker-taiga-example
+#
+# Please modify this file as needed, see the local.py.example for details:
+# https://github.com/taigaio/taiga-back/blob/master/settings/local.py.example
+#
+# Importing docker provides common settings, see:
+# https://github.com/benhutchins/docker-taiga/blob/master/docker-settings.py
+# https://github.com/taigaio/taiga-back/blob/master/settings/common.py
+
+from .docker import *
+from ldap3 import Tls
+import ssl
+
+#########################################
+## LDAP
+#########################################
+
+if os.getenv('TAIGA_ENABLE_LDAP').lower() == 'true':
+    # see https://github.com/Monogramm/taiga-contrib-ldap-auth-ext
+    print("Taiga contrib LDAP Auth Ext enabled", file=sys.stderr)
+    INSTALLED_APPS += ["taiga_contrib_ldap_auth_ext"]
+
+    if os.getenv('TAIGA_LDAP_USE_TLS').lower() == 'true':
+        # Flag to enable LDAP with STARTTLS before bind
+        LDAP_START_TLS = True
+        LDAP_TLS_CERTS = Tls(validate=ssl.CERT_NONE, version=ssl.PROTOCOL_TLSv1, ciphers='RSA+3DES')
+    else:
+        LDAP_START_TLS = False
+
+    # Full DN of the service account.
+    # Use to connect to LDAP server and search for login user's account entry
+    # If LDAP_BIND_DN is not specified or blank, then an anonymous bind is attempated
+    LDAP_SERVERS = os.getenv('TAIGA_LDAP_SERVERS')
+    LDAP_PORT = int(os.getenv('TAIGA_LDAP_PORT'))
```

Comment:
@AminovE99 Instead of replacing `SERVER` by `SERVERS` and thus breaking setup of all Taiga environments which do not update their config, you should offer an automatic conversion or at least keep compatible.